### ISSUE TITLE
Creating Character Error

### ISSUE THREAD

**Issue by Ofelin** [AionGermany/aion-germany#9]
Greetings,

When using the 64 bit client, you're unable to create a new character (by clicking the UI button). The 32 bit client works fine. I have tried with both NA and EU client. Also tested with JDK 7 and 8, same results. The error is attached below.

```
14:09:53.824 ERROR[ReadWrite-0 Dispatcher]: - Reading failed for packet [C] 0x153 CM_CREATE_CHARACTER
java.lang.IllegalArgumentException: There is no player class with id -7
        at com.aionemu.gameserver.model.PlayerClass.getPlayerClassById(PlayerClass.java:96) ~[AL-Game.jar:na]
        at com.aionemu.gameserver.network.aion.clientpackets.CM_CREATE_CHARACTER.readImpl(CM_CREATE_CHARACTER.java:115) ~[AL-Game.jar:na]
        at com.aionemu.commons.network.packet.BaseClientPacket.read(BaseClientPacket.java:98) ~[al-commons.jar:na]
        at com.aionemu.gameserver.network.aion.AionConnection.processData(AionConnection.java:297) [AL-Game.jar:na]
        at com.aionemu.commons.network.Dispatcher.parse(Dispatcher.java:245) [al-commons.jar:na]
        at com.aionemu.commons.network.Dispatcher.read(Dispatcher.java:206) [al-commons.jar:na]
        at com.aionemu.commons.network.AcceptReadWriteDispatcherImpl.dispatch(AcceptReadWriteDispatcherImpl.java:80) [al-commons.jar:na]
        at com.aionemu.commons.network.Dispatcher.run(Dispatcher.java:107) [al-commons.jar:na]
```

**Comment by penguinjoe** [comment]
I have seen this as well and commented on it (i si8nce closed the issue as I felt it may be unfixable). I suspect the client is aligning ints a little differently than the 32-bit client for the CM_CREATE_CHARACTER packet. The only way to be sure is to capture and dump the inbound CM_CREATE_CHARACTER packets from the client and compare them to see where the extra padding is placed.

**Comment by FrozenKiller35** [comment]
@Ofelin 

Already known but no fix yet and thats why we use and support 32 bit only.
But as you know you can create a char with 32Bit and then play with 64Bit ^^

Oh and THX for the Report ;)

**Comment by Ofelin** [comment]
Hehe, i am glad you guys took your time to comment on this. Well, it might be fixed when/if the project moves to 6.0 development!! :P

**Comment by penguinjoe** [comment]
Hmm, looking at that I see that the al_commons jar does some parsing on it. Perhaps I will retry building an updated version of that jar and see if I can find where things go sideways in that packet.
